### PR TITLE
added bio for Drew 

### LINKED ISSUE #3: <andrew_gautier> Onboarding

### PR DESCRIPTION
**Pull request by Gautier404**
closes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +13,7 @@ Get off the cross, we need the wood.
 ### Tyler McCown -- Controls Grandpappy
 
 You better settle down over there or I'm gonna come talk to you.
+
+### Drew gautier -- New Member
+
+Mechanical engineer from Novato, California who is wondering why Tommy is so weird.
```
Comment (thomaskost17) [inline README.md]: lmao

**Review by thomaskost17 — COMMENTED** [review]
looks like youre missing the file in the practice Files directory. Would like to see that in the PR before it gets accepted. 

looking like going well tho.

**Comment by Gautier404** [comment]
Oh sorry I thought you wanted us to not include the practice file in the pull request.

**Comment by thomaskost17** [comment]
> Oh sorry I thought you wanted us to not include the practice file in the pull request.

no problem, when you add it ill close the PR

**Comment by thomaskost17** [comment]
noticed you added the files, everything looks great. Not to be nit picky but if you could squash the commits and push with a `git push --force` that'd be great

**Comment by thomaskost17** [comment]
Looks good to me

**Pull request merged**